tests/Doctrine/DBAL/Migrations/Tests/ConfigurationTest.php

```diff
@@ -191,6 +192,31 @@ public function testFormatVersion($version, $return)
     }
 
     /**
+     * @dataProvider autoloadVersionProvider
+     *
+     * @param $version
+     */
+    public function testGetVersion($version)
+    {
+        $config = $this->getSqliteConfiguration();
+        $config->setMigrationsNamespace('Doctrine\DBAL\Migrations\Tests\Stub\Configuration\AutoloadVersions');
+        $config->setMigrationsDirectory(__DIR__ . '/Stub/Configuration/AutoloadVersions');
+
+        $result = $config->getVersion($version);
+
+        $this->assertNotNull($result);
+    }
+
+    public function testGetVersionNotFound()
+    {
+        $this->setExpectedException(MigrationException::class);
```

Comment:
Putting this after the call to ``getSqliteConfiguration`` would be even better, as it would enforce the exception to happen in ``getVersion`` rather than ``getSqliteConfiguration`` (which is the drawback of the ``@expectedException`` annotation btw)